src/main/java/se/sundsvall/templating/domain/ContextMunicipalityId.java

```diff
@@ -0,0 +1,10 @@
+package se.sundsvall.templating.domain;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ContextMunicipalityId {
+	private String id;
```

Comment:
The class is named ContextMunicipalityId and wraps a municipalityId.

Rename `id `to `value`?